pkg/securitycontext/util.go

```diff
@@ -124,6 +124,22 @@ func DetermineEffectiveSecurityContext(pod *v1.Pod, container *v1.Container) *v1
 	return effectiveSc
 }
 
+// DetermineEffectiveRunAsUser returns a UID from the provided pod's and
+// container's security context. Container's runAsUser take precedence in cases
+// where both are set.
+func DetermineEffectiveRunAsUser(pod *v1.Pod, container *v1.Container) *int64 {
```

Comment:
```suggestion
func DetermineEffectiveRunAsUser(pod *v1.Pod, container *v1.Container) (int64, bool) {
```

https://github.com/golang/go/wiki/CodeReviewComments#in-band-errors

Reply:
done